arm-containerregistry/2016-06-27-preview/swagger/containerregistry.json

```diff
@@ -488,7 +488,8 @@
         },
         "location": {
           "description": "The location of the resource. This cannot be changed after the resource is created.",
-          "type": "string"
+          "type": "string",
+          "readOnly": true

```

Comment:
Good point. I will revert the change made in this commit, I thought it was an oversight at first.